Source/ARTLocalDevice.m

```diff
@@ -94,7 +94,7 @@ - (void)setDeviceToken:(NSString *_Nonnull)token {
 }
 
 - (void)setAndPersistDeviceToken:(NSString *)deviceToken {
```

Comment:
I know you didn’t change it but it should be called `setAndPersistAPNSDeviceToken`

Reply:
done f9c88fd